src/main/java/uk/gov/companieshouse/accounts/filing/configuration/ApplicationConfiguration.java

```diff
@@ -22,4 +23,13 @@ public Logger logger() {
         return LoggerFactory.getLogger(applicationNameSpace);
     }
 
+    /**
+     * Creates the internal user interceptor used by the application.
+     *
+     * @return the internal user interceptor
+     */
+    @Bean
+    public InternalUserInterceptor internalUserInterceptor() {return new InternalUserInterceptor(applicationNameSpace);
+    }
```

Comment:
The style is a little odd here should be: 
```suggestion
    public InternalUserInterceptor internalUserInterceptor() {
        return new InternalUserInterceptor(applicationNameSpace);
    }
```